packages/@react-native-windows/telemetry/src/test/projects/UsesPackageReference/UsesPackageReference.csproj

```diff
@@ -16,7 +16,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```

Comment:
calling this out that this will require updating the documentation that we will drop support for 16299;
are our partners ok with this?

@stmoy @chrisglein fyi